site/content/docs/5.3/components/collapse.md

```diff
@@ -126,15 +135,17 @@ These classes can be found in `_transitions.scss`.
 
 Just add `data-bs-toggle="collapse"` and a `data-bs-target` to the element to automatically assign control of one or more collapsible elements. The `data-bs-target` attribute accepts a CSS selector to apply the collapse to. Be sure to add the class `collapse` to the collapsible element. If you'd like it to default open, add the additional class `show`.
 
-To add accordion-like group management to a collapsible area, add the data attribute `data-bs-parent="#selector"`. Refer to the [accordion page]({{< docsref "/components/accordion" >}}) for more information.
+To add accordion-like group management to a collapsible area, add the data attribute `data-bs-parent="#selector"`. Refer to the [Accordion page](https://getbootstrap.com/docs/5/components/accordion/) for more information.
```

Comment:
This link is already OK (see https://twbs-bootstrap.netlify.app/docs/5.3/components/collapse/#via-data-attributes). Moreover, your modification  breaks it completely (see https://deploy-preview-38147--twbs-bootstrap.netlify.app/docs/5.3/components/collapse/#via-data-attributes) since it links to https://getbootstrap.com/docs/5/components/accordion/ which doesn't exist.